instrumentation/oshi/library/src/main/java/io/opentelemetry/instrumentation/oshi/ProcessMetrics.java

```diff
@@ -21,8 +22,10 @@ private ProcessMetrics() {}
 
   /** Register observers for java runtime metrics. */
   public static void registerObservers() {
-    // TODO(anuraaga): registerObservers should accept an OpenTelemetry instance
```

Comment:
Thanks for helping on this! Let's remove these methods and have the callers pass in a opentelemetry. The javaagent will pass in GlobalOpenTelemetry.get and library instrumentation tests will pass in `testing.getOpenTelemetry()` if I'm not mistaken

Reply:
Should I add `@Deprecated` to the methods first to give time for users to update their apps and then in a different release we remove them?

Reply:
> Should I add `@Deprecated` to the methods first to give time for users to update their apps and then in a different release we remove them?

that would be great 👍

Reply:
oh, I see you did already, thx!